<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolButton" name="browseButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>221</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="filePath">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>220</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="convert">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>470</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert and Save</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>621</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QRadioButton" name="image2pdf">
     <property name="text">
      <string>Image2pdf</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="word2pdf">
     <property name="text">
      <string>word2pdf</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="pptx2pdf">
     <property name="text">
      <string>pptx2pdf</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>185</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Input files</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="outputFileNameBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>360</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>330</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Output file name</string>
    </property>
   </widget>
   <widget class="QToolButton" name="outputFolderBrowse">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>360</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Output Folder</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
